### PR TITLE
Chore/documentation fixes

### PR DESCRIPTION
**Pull request by davidweterings**

### CODE REVIEW

File: config/types_azure.go [hunk 1]
```diff
@@ -174,9 +174,9 @@ type AzureFrontdoorSettings struct {
 	DnsResourceGroup string                   `yaml:"dns_resource_group"`
 	SslKeyVault      *AzureFrontdoorSslConfig `yaml:"ssl_key_vault"`
 
-	// Undocumented option to workaround some tenacious issues
+	// Undocumented option to work around some tenacious issues
 	// with using Frontdoor in the Azure Terraform provider
-	SupressChanges bool `yaml:"supress_changes"`
+	SuppressChanges bool `yaml:"suppress_changes"`
```
Comment (mvantellingen) [inline config/types_azure.go:179]: This does introduces a backwards incompatible change: https://github.com/labd/mach-composer/blob/2ec4c78c4943b37aab684b0d56a7292f7444e1da/config/types_azure.go#L179

But i'm fine with it

File: runner/hash.go [hunk 2]
```diff
@@ -30,12 +30,12 @@ func GetHash(path string) string {
 		if err != nil {
 			panic(err)
 		}
-		defer f.Close()
 
 		if _, err := io.Copy(h, f); err != nil {
 			log.Fatal(err)
 		}
 
+		f.Close()
```
Comment (mvantellingen) [inline runner/hash.go:38]: Don't we want to use defer here? If the io.Copy() fails we don't close the input file now

File: config/types_azure.go (hunk 1 above)
Comment (davidweterings) [reply config/types_azure.go:179]: Actually the documentation already says it should be what I changed it into; https://docs.machcomposer.io/reference/syntax/global.html#frontdoor

File: runner/hash.go (hunk 2 above)
Comment (davidweterings) [reply runner/hash.go:39]: Fixed by explicitly closing there too. Not great but it closes the technically incorrect defer

File: config/types_azure.go (hunk 1 above)
Comment (tleguijt) [reply config/types_azure.go:179]: Indeed, was also implemented like that 
afbeelding

**Review by tleguijt — APPROVED** [review]

**Pull request merged**